src/pages/patch/buildVariants/GroupedTaskSquare.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { Tooltip } from "antd";
+import queryString from "query-string";
+import { useParams } from "react-router-dom";
+import { usePatchAnalytics } from "analytics";
+import { StyledRouterLink } from "components/styles";
+import { getVersionRoute } from "constants/routes";
+
+interface Props {
+  count: number;
+  statuses: string[];
+  color: string;
+  textColor: string;
+}
+
+export const GroupedTaskSquare: React.FC<Props> = ({
+  statuses,
+  count,
+  color,
+  textColor,
+}) => {
+  const patchAnalytics = usePatchAnalytics();
+  const { id } = useParams<{ id: string }>();
+
+  const nextQueryParams = queryString.stringify(
+    {
+      statuses,
+      page: 0,
+    },
+    { arrayFormat }
+  );
+  const status = "test";
```

Comment:
This looks mostly unused now, but I think we now want to send the color in the event right?

Reply:
Nice catch!